recipe/meta.yaml

```diff
@@ -44,3 +46,4 @@ extra:
   recipe-maintainers:
     - frol
     - isuruf
+    - inducer
```

Comment:
FYI. View this as an offer, happy to revert.

Reply:
I’m happy to pass it over! 🙏